handbook/company.md

```diff
@@ -78,27 +78,27 @@ Empathy leads to better understanding, better communication, and better decision
   - Own up to mistakes.
   - Understand why it matters (the goals of the work you are doing).
   - Consider the business impact (fast forward 12 months, consider the total cost of ownership over the eternity of maintenance).
-  - Do things that don't scale, sometimes.
+  - Do things that don't scale, sometimes. 
```

Comment:
@mikermcneil, could you give me an example of this?

Reply:
For example, before investing in automation, start by doing a process manually, by hand.  Even if it won't scale months into the future, it will be good enough for now, come to life quickly, and help inform what _actually_ needs to happen later, at scale, based on the experience of having done it first by hand.

Reply:
@Desmi-Dizney For example, before investing in automation, start by doing a process manually, by hand. Even if it won't scale months into the future, it will be good enough for now, come to life quickly, and help inform what actually needs to happen later, at scale, based on the experience of having done it first by hand.

Reply:
I think that this is helpful context. How does this sound?

"Occasionally, you'll need to own processes that won't scale. Not everything can be automated from the start, but your experience with doing things manually will teach us how to scale later."

Reply:
@Desmi-Dizney This is good.  I think I'd replace "Occasionally" with "Often".

Reply:
Also, it's not that everything _can't_ be automated from the start.  It's that if we try to do that without going into the trenches and doing it by hand, ourselves, we wouldn't know how to automate it the best way for Fleet.  i.e. premature automation.

After doing something by hand a few times, you learn it well enough to be able to successfully scale/automate it.  Otherwise, chances are, you'll scale/automate in a way that won't _quite_ work, which leads to even more work.

Reply:
@Desmi-Dizney Famous essay on the subject: http://paulgraham.com/ds.html

Reply:
@mikermcneil, how does this look?

"Often, you'll need to own processes that won't scale. Not everything should be automated from the start. Your experience with doing things manually will teach us how to scale effectively later."

Reply:
Done